src/dark.css

```diff
@@ -119,4 +119,6 @@ img { background: white; }
 	--outdated-shadow: red;
 
 	--editedrec-bg: darkorange;
+
+	--uname: rgb(215 99 99);
```

Comment:
```suggestion
	--uname: rgb(215,99,99);
```

Reply:
@xfq how did you come up with that color?

I checked the contract ratio and it [doesn't pass WCAG AAA](https://color.a11y.com/ContrastPair/?bgcolor=000&fgcolor=D76363).

[rgb(215, 140, 140)](https://color.a11y.com/ContrastPair/?bgcolor=000&fgcolor=D78C8C) seems to work .better

Reply:
I selected it manually, because I think the contrast is good enough for me and AAA requirements are not recommended to be included in legislation by default, but I'm OK with colour you picked as well.

As for the syntax, since the recommended [space-separated notation](https://www.bram.us/2020/04/27/colors-in-css-hello-space-separated-functional-color-notations/) is now supported [by all major browsers](https://caniuse.com/mdn-css_types_color_rgb_space_separated_parameters), I wonder if we can adopt it?